test/Polytopes/Polyhedron.jl

```diff
@@ -11,6 +11,8 @@
     square = cube(2)
     C1 = cube(2, 0, 1)
     Pos = Polyhedron([-1 0 0; 0 -1 0; 0 0 -1], [0,0,0])
+    # TODO:
+    # @test Polyhedron(([-1 0 0; 0 -1 0; 0 0 -1], [0,0,0]); non_redundant = true) == Pos
```

Comment:
```suggestion
    @test Polyhedron(([-1 0 0; 0 -1 0; 0 0 -1], [0,0,0]); non_redundant = true) == Pos
```
If I read through this correctly the corresponding constructor should be available, so why is this commented? (If It fails for some reason we could also use `@test_broken`?)

Reply:
This depends on the far facet and how to deal with that will be discussed in a separate issue.